nextflow_schema.json

```diff
@@ -233,16 +240,12 @@
             "default": "",
             "fa_icon": "fas fa-toolbox",
             "properties": {
-                "joint_germline": {
-                    "type": "boolean",
-                    "fa_icon": "fas fa-align-justify",
-                    "description": "Enables GATK4 joint germline variant calling, if also haplotypecaller is selected"
-                },
-                "ascat_ploidy": {
+                "ploidy": {
                     "type": "number",
-                    "fa_icon": "fas fa-wrench",
-                    "description": "Overwrite ASCAT ploidy",
-                    "help_text": "Requires that `--ascat_purity` is set"
+                    "fa_icon": "fas fa-bacon",
```

Comment:
Love the bacon icon for ploidy, good thinking

Reply:
why don't they have chromsomes. Never realized the bacon icon is also used for conda 😆